Javad size control with range control

javadsamiee opened the pull request.

javadhub committed:
AlignmentControl label option

javadhub committed:
Add RangeControl
Change Column Maxi 3 tabs to 2 tabs
Change black background of block sidebar settings title

javadhub committed:
fix styles

javadhub committed:
Move rangeControl into SizeControl
Add showRangeControl property to SizeControl

Naaaaiix reviewed (commented):
One last comment:
- Avoid deprecated blocks and components. On next update I'll remove them, are useless

javadhub committed:
Replace " with '
Remove showRangeControl property and show it always
Remove Fragment from sizeControl

Naaaaiix reviewed (approved):
Perfect, good job @javadsamiee ! 😄
